### ISSUE TITLE
weather.com - functional fracture

### ISSUE THREAD

**Issue by Rusenche** [uBlockOrigin/uAssets#19399]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

Activating the filter: https://easylist-downloads.adblockplus.org/cntblock.txt breaks part of the site's functionality.

### URL(s) where the issue occurs.

https://weather.com/bg-BG/weather/today/l/1c66dfadc4ae5be6fafa98d84be321d66db0daf598b0314a7d44f08d02ece7f1

![HhH](https://github.com/uBlockOrigin/uAssets/assets/18578128/324c7f25-f177-4a10-9aa2-e290cc0800e4)

### Screenshot(s)

![HhH](https://github.com/uBlockOrigin/uAssets/assets/18578128/1125607c-0a1e-4a41-bd57-b4e456888569)

### uBO version

1.50.0

### Browser name and version

Ungoogled Chromium v.115.0.5790.131 x64

### Settings

I enabled the filter: https://easylist-downloads.adblockplus.org/cntblock.txt because of the yandex.com site, but that broke the functionality of weather.com.

When I disable RU AdList: Counters - then weather.com functionality is restored.

### Notes

Activating the filter: https://easylist-downloads.adblockplus.org/cntblock.txt breaks part of the site's functionality.

I enabled the filter: https://easylist-downloads.adblockplus.org/cntblock.txt because of the yandex.com site, but that broke the functionality of weather.com. What I show in the picture - it is not possible to load.

When I disable RU AdList: Counters - then weather.com functionality is restored.

![HhH](https://github.com/uBlockOrigin/uAssets/assets/18578128/015dec96-cfcc-4c2b-a608-0d980c899239)

**Comment by dportvine** [comment]
@dimisa-RUAdList

**Comment by dimisa-RUAdList** [comment]
I can't reproduce this problem, but I'll get back to it tomorrow.

**Comment by Rusenche** [comment]
> I can't reproduce this problem, but I'll get back to it tomorrow.

Right now no one can either tomorrow or the day after tomorrow because the chance of rain (in 1 hour) is already gone and that window is gone too... It's just that this had to be tested now, not days later...

The alternative is to look for a city where there is such a forecast for probable precipitation, but...

**Comment by Rusenche** [comment]
I tested now with:
- Ungoogled Chromium v.116.0.5845.97;
- uBlock Origin v.1.51.0;
- completely new profiles.

Everything is just with its standard settings - the browser and uBlock Origin.

How to reproduce another problem I just found?!

1. Add this filter https://easylist-downloads.adblockplus.org/cntblock.txt to uBlock Origin v.1.51.0.
2. Refresh all filters.
3. Visit https://weather.com/.
4. Try typing something - anything in the search box.

There is no chance of typing in the search box of https://weather.com/.

And now - turn off filter https://easylist-downloads.adblockplus.org/cntblock.txt from uBlock Origin v.1.51.0.

Voila - you can type in the https://weather.com/ search box.

![HJAjha](https://github.com/uBlockOrigin/uAssets/assets/18578128/b5d4b38a-3443-4142-815c-fbfbb1a14a11)

**Comment by stephenhawk8054** [comment]
@Rusenche Can you test this filter?

```adb
@@||weather.com/api/v1/script/dprSdkScript.js$script,1p
```

**Comment by Rusenche** [comment]
> @Rusenche Can you test this filter?
> 
> ```
> @@||weather.com/api/v1/script/dprSdkScript.js$script,1p
> ```

I can confirm that this custom filter works successfully for the second issue - it's now typeable in the site's search box.

But apparently the filter list itself - https://easylist-downloads.adblockplus.org/cntblock.txt needs to be added or edited so it doesn't harm weather.com.


No offense, but this is a weather forecast site, not a news site. Therefore the weather forecast is changing rapidly and can no longer check the stated problem at the moment regarding precipitation (I mean the one surrounded by a red rectangle - can't load at all) - this from the first picture of the first opinion...


![261784683-324c7f25-f177-4a10-9aa2-e290cc0800e4](https://github.com/uBlockOrigin/uAssets/assets/18578128/fb10843d-1ac2-4944-a2d5-52f766ff13e8)

**Comment by stephenhawk8054** [comment]
No filter lists are immune to false positives, of course. Filter lists are improved by users' reports and feedbacks. They aren't created to work 100% correctly magically.

Thing is, in order for filter lists maintainers to fix the issues, they have to reproduce the issue themselves, otherwise how can they fix it?

**Comment by Rusenche** [comment]
> Thing is, in order for filter lists maintainers to fix the issues, they have to reproduce the issue themselves, otherwise how can they fix it?

This is the key point. As in life - there is always a "BUT". The particular issue is specific because a separate window appears in supposedly bad weather conditions. In this particular case, it was supposed to be rain, which actually turned out to be a false forecast. The problem in this particular case is that it is no longer possible to reproduce the problem - the weather has changed and the sun is already shining. Therefore - the window that I showed in the picture will not appear. That's why I wrote that the specific problem is specific. Because it's not a news site, a file download site, or a sports site where everything is in its place or buried on a back page like old news.

This problem will remain unsolved because it can't be reproduced anymore, the weather forecast has already changed and the window I showed in a picture will not appear (until the next rain forecast for example).

But that's what users are for - to report problems and have the corresponding filter lists corrected as perfect as possible.

**Comment by stephenhawk8054** [comment]
Yes, maintainers understand that. And it's better to focus on how to help them reproduce the issue instead of mentioning off-topic subjects.

**Comment by dimisa-RUAdList** [comment]
Fix: https://github.com/easylist/ruadlist/commit/f30c0b17037724e845e002669ca9420e7e5a82db

**Comment by Rusenche** [comment]
I have currently tested the fix in Podgorica and in Warsaw. It works.

Thanks.
